package.json

```diff
@@ -37,7 +51,8 @@
     "lodash.escaperegexp": "^4.1.2",
     "node-fetch": "^2.6.7",
     "prettier": "^3.4.2",
-    "semver": "^7.7.2"
+    "semver": "^7.7.2",
+    "typescript": "^5.9.2"
```

Comment:
I doubt if this mismatched typescript version in the root level is causing some ci failures, in main-branch no errors in e2e-tests or lint errors which is observed in this PR.